install/kubernetes/helm/istio/charts/pilot/templates/clusterrole.yaml

```diff
@@ -28,6 +28,12 @@ rules:
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: ["create", "get", "list", "watch", "update"]
+- apiGroups: [""]
```

Comment:
I'm a bit scared of this - can you add a normal role ( == restricted to pilot's namespace ) and only grant secret reading with a role binding ? Pilot should not have privs to read cluster secrets.

Reply:
@costin when shared informer is used, there is no choice, it watches in all namespaces. If you insist fixing it in this PR, I can do it, but it would require a bit of time. If it can be done in a following PR, I promise to do it right after.

Reply:
I don't think there is a particular reason to hurry this in - we can cherry pick next week if it takes longer, and it is not blocking the testing. It's a very nice usability improvement and happy to see it, but no reason to hurry.